narayana-release-process.sh

```diff
@@ -178,8 +105,13 @@ then
 fi
 
 rm -rf $PWD/localm2repo
+# uploaded artifacts go straight live without the ability to close the repo at the end, so the install is done to verify that the build will work
```

Comment:
@marcosgopen it seems you added this comment to the install for the documentation, rather than the install for narayana - please can you take a look?

Reply:
Thanks Tom, I made a mistake! I will update the PR.